sound/soc/sof/imx/imx8m.c

```diff
@@ -315,6 +326,39 @@ static struct snd_soc_dai_driver imx8m_dai[] = {
 		.channels_max = 32,
 	},
 },
+{
```

Comment:
"There is no SAI4" would be a highly unusual way of designing hardware?

The first Google search gives me this
![image](https://github.com/thesofproject/linux/assets/1149035/6b29d3c9-d180-4aed-89cc-a6ae0cd9d2c7)

Reply:
I suspect that NXP won't support HDMI in Sound Open Firmware, so skipping it is likely fine here. If that changes at some point then fair game though.

Reply:
The use of imx8m in SOF is a bit unfortunate (or I just don't understand it) because NXP i.MX8M does not even have a DSP. So it is more about i.MX8MPlus... 
imx8 and imx8ulp don't have a SAI 4 as far as I saw. They do have more SAIs than are defined in imx8.c and imx8ulp.c but I concerned myself only with imx8mp.